django/thunderstore/api/cyberstorm/tests/test_community.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from django.db.models import Count, Sum
+from rest_framework.test import APIClient
+from thunderstore.community.models.community import Community
+
+from thunderstore.community.models import CommunitySite
+
+
+@pytest.mark.django_db
+def test_views_communities(client: APIClient, community_site: CommunitySite):
```

Comment:
More tests could be added, e.g. unlisted communities, community with no packages or lots of packages, none/lots of downloads, has image/no image... This comment applies to other test files in this PR too.

Also I kinda like the naming convention that goes something like theThingToBeTested_stateToBeTested_expectedBehaviour. E.g. something like `test_communities_view__with_valid_request__returns_valid_data` here. (It's a bit silly example since this is a basic case with success response, but makes more sense when there's more tests that test specific scenarios.)

Reply:
> More tests could be added, e.g. unlisted communities, community with no packages or lots of packages, none/lots of downloads, has image/no image... This comment applies to other test files in this PR too.

I've tried to add pretty well rounded tests now. And think these will be sufficient for now. There are improvements that can be made, but I'd rather not use much more time on these tests, as this will be an "beta" endpoint for now.

> Also I kinda like the naming convention that goes something like theThingToBeTested_stateToBeTested_expectedBehaviour. E.g. something like `test_communities_view__with_valid_request__returns_valid_data` here. (It's a bit silly example since this is a basic case with success response, but makes more sense when there's more tests that test specific scenarios.)

I haven't used this naming convention that much, but I've renamed many of my now added tests to adhere to this naming convention.

Reply:
I've added multiple sets of tests that should cover a pretty wide area of usage. Though if you do spot something in the new split PRs (or in this branch) feel free to point it out 👍